### PR TITLE
Renovate issue templates

### PR DESCRIPTION
**Pull request by Uzay-G**
Part of [plots2/7141](https://github.com/publiclab/plots2/issues/7141).
Image-sequencer was not using the new github template setup. Now, contributors will be prompted to pick one of the templates when they create an issue. The issue process will be like this for the repo:
template-setup

### CODE REVIEW

**Comment by blurry-x-face** [comment]
This is a nice update!!

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+
+
+> What did you expect to see that you didn't?
+
+
+
+### Please show us where to look
+
+https://beta.sequencer.publiclab.org
```
Comment (jywarren) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Can we offer some options here too, like https://sequencer.publiclab.org and even localhost?

**Comment by SidharthBansal** [comment]
@Uzay-G kindly complete it asap. As soon as @HarshKhandeparkar will approve it I will merge it.
This is again 1 point task

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+### What's your PublicLab.org username?
```
Comment (HarshKhandeparkar) [inline .github/ISSUE_TEMPLATE/bug-report.md]: I don't this line is required in IS.

**Comment by HarshKhandeparkar** [comment]
@Uzay-G are you still working on this? Do you need any help? If you don't have time, then I can continue this PR and open a new one.

**Comment by blurry-x-face** [comment]
@Uzay-G are you still working on this?

**Comment by HarshKhandeparkar** [comment]
@jywarren I think this one can be merged with a few changes (if it is possible for you to add those)

**Comment by jywarren** [comment]
Oh no I meant to reopen to retrigger the build but it won't let me reopen!

**Pull request closed without merging**